eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/tcp/client/group/ClientSessionGroupMapping.java

```diff
@@ -260,8 +260,7 @@ private void startClientGroupConsumer(Session session) throws Exception {
         }
         synchronized (lockMap.get(subsystem)) {
             log.info("readySession session[{}]", session);
-            ClientGroupWrapper cgw = session.getClientGroupWrapper().get();
-
+            ClientGroupWrapper cgw = this.getClientGroupMap().get(session.getClient().getGroup());
             boolean flag = cgw != null && cgw.addGroupConsumerSession(session);
```

Comment:
The key of `clientGroupMap` is `subsystem`, I'm not sure whether the `group` key here works well.